youtube_dl/extractor/wikimedia.py

```diff
@@ -0,0 +1,67 @@
+from .common import InfoExtractor
+from ..utils import get_element_by_class, determine_ext, clean_html, KNOWN_EXTENSIONS
+import re
+
+
+class WikimediaIE(InfoExtractor):
+    _NETRC_MACHINE = 'wikimediaorg'
+    IE_NAME = 'wikimedia.org'
+    _API_BASE_URL = 'https://commons.wikimedia.org/'
+    _VALID_URL = r'https://commons.wikimedia.org/wiki/File:(?P<id>[^/]+)'
+
+    _TEST = {
+        'url': 'https://commons.wikimedia.org/wiki/File:Die_Temperaturkurve_der_Erde_(ZDF,_Terra_X)_720p_HD_50FPS.webm',
+        'info_dict': {
+            'description': 'Deutsch:  Beschreibung auf der Seite: "Im Verlauf der Erdgeschichte glich das Klima einer Achterbahnfahrt. Die „Fieberkurve“ unseres Planeten zeigt die globalen Temperaturschwankungen bis heute – rekonstruiert anhand von historischen Klimadaten."\nZu Wikimedia Commons hochgeladen von: PantheraLeo1359531.\nHinweise zur Weiterverwendung: https://www.zdf.de/dokumentation/terra-x/terra-x-creative-commons-cc-100.html.\nVereinfachender Verlauf in der Geschichte der Erde, für die Zukunft spätestens ab dem Jahr 2050 mit spekulativem Verlauf in der Prognose (ausgeprägtes Global-warming-Szenario ist dargestellt).English:  Climate change, Temperature in history of Earth, Video of Terra X.',
```

Comment:
you should use md5 for long fields like this
```py
            'description': 'md5:xxx',
```

Reply:
done